imod/mf6/model_gwf.py

```diff
@@ -330,6 +330,9 @@ def from_imod5_data(
                     wel_key, imod5_data, times
                 )
 
+        if "cap" in imod5_keys:
+            result["msw-rch"] = Recharge.from_imod5_cap_data(imod5_data)  # type: ignore
```

Comment:
I would reconsider the place you put you converter code. The Recharge package now needs to now about the imod5 model. What would happen if we add another converter e.g. flopy? Then mf6 module also needs to know about that.

I would recommend moving all the converter code to a separate module. And in that module the code then knows about the imod5 data/packages and the mf6 data/packages

Reply:
I agree, but it would be too large of a task to fix in this PR. I created this issue for this: https://github.com/Deltares/imod-python/issues/1319